code/centauri/algorithms/grandpa/prover/build.rs

```diff
@@ -15,6 +15,7 @@
 
 #[tokio::main]
 async fn main() -> color_eyre::Result<()> {
-	subxt_codegen::build_script("ws://127.0.0.1:9944", "runtime").await?;
+	subxt_codegen::build_script("ws://127.0.0.1:9944", "polkadot").await?;
+	subxt_codegen::build_script("ws://127.0.0.1:9188", "parachain").await?;
 	Ok(())
 }
```

Comment:
build.rs will make deterministic builds a lot more difficult; so at some point we'll remove this and generate this stuff with nix.

Reply:
actually it is blocker for having deterministic(nix) hyperspace and docker compose builds [docker](https://app.clickup.com/t/3p6g7gr). so we cannot help bridge team until that is fixed. i would prefer CI nix and local dev would follow same flow. so in this case build.rs will be removed. also build.rs will fail to maintain several relays version (non local) and runtimes.  cc @seunlanlege

Reply:
cc @andor0 @cor